scripts/install/installnode.py

```diff
@@ -0,0 +1,512 @@
+                "maxram": "100",
+                "disk": "0",
+                "unixsocket": "0"
+            },
+            'servicename': 'redis',
+            'instance': 'system',
+            'daemonpath': '/usr/bin/redis-server',
+            'args': '/opt/jumpscale7/var/redis/redis__system/redis.conf',
+            'pwd': '/opt/jumpscale7'
+        }
+
+        # self.redis2 = {
+        #     'servicename': 'redis',
+        #     'instance': 'server',
+        #     'daemonpath': '/usr/bin/redis-server',
+        #     'args': '/etc/redis/redis.conf',
+        #     'pwd': '/opt/jumpscale7'
+        # }
+
+        self.jsagent = {
+            'config': {
+                "agentcontroller.connection": "main",
+                "grid.id": str(self.config.gid),
+                "grid.node.roles": ",".join(self.config.roles),
+                "osis.connection": "jsagent",
+            },
+            'servicename': 'jsagent',
+            'daemonpath': '/usr/local/bin/jspython',
+            'args': 'jsagent.py -i main',
+            'pwd': "/opt/jumpscale7/apps/jsagent/",
+            'instance': 'main',
+            'domain': 'jumpscale',
+        }
+
+        self.osis_client_main = {
+            "config": {
+                "instance.param.osis.client.addr": self.config.masterips,
+                "instance.param.osis.client.login": "root",
+                "instance.param.osis.client.passwd": self.config.password,
+                "instance.param.osis.client.port": "5544",
+                },
+                "domain": 'jumpscale',
+                "servicename": "osis_client",
+                "instance": "main"
+            }
+        import copy
+        self.osis_client_jsagent = copy.deepcopy(self.osis_client_main)
+        self.osis_client_jsagent.update({'instance': 'jsagent'})
+
+        self.agent_controller_client = {
+            "config": {
+                "instance.agentcontroller.client.addr": self.config.masterips,
+                "instance.agentcontroller.client.login": "node",
+                "instance.agentcontroller.client.passwd": "",
+                "instance.agentcontroller.client.port": "4444",
+                },
+                "domain": 'jumpscale',
+                "servicename": "agentcontroller_client",
+                "instance": 'main'
+            }
+
+    def install(self):
+        self.install_packages()
+        self.install_redis()
+        self.install_osis_client()
+        self.install_agentcontroller_client()
+        self.install_jsagent()
+        # self.install_nginx()
+    
+    def install_redis(self):
+        config = self.redis['config']
+        j.clients.redis.configureInstance(
+            "{}__{}".format(self.redis['servicename'], self.redis['instance']),
+            ip=config['ip'],
+            port=config['port'],
+            maxram=config['maxram'],
+            appendonly=False,
+            unixsocket=config['unixsocket']
+            )
```

Comment:
we also need to write a fake service.hrd for this one (it's used by the redis client)